plugin/src/main/resources/poller-configuration.xml

```diff
@@ -1,6 +1,6 @@
 <poller-configuration xmlns="http://xmlns.opennms.org/xsd/config/poller/api">
     <package name="velocloud">
-        <filter>IPADDR != '0.0.0.0'</filter>
+        <filter>IPADDR != '255.255.255.255'</filter>u
```

Comment:
I really do not like this. Maybe we should pick the first interface or the first public IP or something like that.

Reply:
After some discussion, we figured that this is the best way to do this.